client/src/pages/Create.js

```diff
@@ -29,35 +32,56 @@ const Location = styled.div`
   }
 `;
 
+const CancelButton = styled(Button)`
+  color: #ffffff;
+  background: #de3a3a;
+`;
+
+const FooterGradient = styled.div`
+  height: 0.4em;
+  background: var(--bg-main-gradient);
+`;
+
+const Footer = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100vw;
+  background: #fff;
+  & button {
+    margin: 0.5em;
+  }
+`;
 export function Create() {
+  const [value, setValue] = useState("misc");
```

Comment:
What is `value`? It's unspecific. It looks like it is used as `category`?

Reply:
Changed it.